int32 pmotorNumber
int32 pmotorSpeed

